run0/src/main.rs

```diff
@@ -149,6 +149,20 @@ impl Runtime {
 
         Ok(())
     }
+
+    fn get_env_variables(&self) -> Result<Vec<(&str, &str)>> {
+        let mut env_variables = Vec::<(&str, &str)>::new();
+
+        if let Some(process) = self.spec.process() {
+            if let Some(env) = process.env() {
+                for var in env {
+                    let key_value = var.split("=").collect::<Vec<&str>>();
+                    env_variables.push((key_value[0], key_value[1]));
+                }
+            }
+        }
+        Ok(env_variables)
```

Comment:
Since we always return a `Vec` and it can be empty, I'd simply return `env_variables` and the method prototype should thus be:

```rust
fn env_variables(&self) -> Vec<(&str, &str)>{
```